src/Val.cc

```diff
@@ -330,7 +330,10 @@ TableValPtr Val::GetRecordFields() {
 }
 
 // This is a static method in this file to avoid including rapidjson's headers in Val.h because
-// they're huge.
+// they're huge. For records, two arguments control the JSON result: only_loggable, if true,
+// skips any fields that don't have the &log attribute. The regular expression matcher, if
+// given, is used to strip the first match on any record field name in the resulting output.
+// See the to_json() BiF for context.
```

Comment:
Moving that comment to `Val::ToJSON()` in `Val.h` would be visible in the headers we ship, too, and potentially more useful.